test/check-application

```diff
@@ -132,8 +130,10 @@ class TestApplication(testlib.MachineCase):
         b.wait_present('.modal-dialog div:contains("io.podman.ErrorOccurred")')
         b.click(".modal-dialog div .alert .close")
         b.wait_not_present(".modal-dialog div .alert")
+        self.confirm_modal("btn-ctr-cancel-commit")
```

Comment:
That looks a bit weird, how does the event handling have an effect on suddenly showing a dialog here?

Reply:
This is totally a wrong paste or something. And it's indeed weird, not sure where that modal did appear from. I removed the line and it succeeds as well so if you don't want further investigation what was going on I will just ignore it.